src/libraries/Microsoft.PowerFx.Core/Public/DisplayNames/DisplayNameProvider.cs

```diff
@@ -24,12 +24,8 @@ public abstract class DisplayNameProvider
         /// If this isn't supported by a given display name provider, this should return the same as 
         /// <see cref="TryGetLogicalName(DName, out DName)"/>, with the newDisplayName output populated by the first arg. 
         /// </summary>
-        internal abstract bool TryRemapLogicalAndDisplayNames(DName displayName, out DName logicalName, out DName newDisplayName);
+        public abstract bool TryRemapLogicalAndDisplayNames(DName displayName, out DName logicalName, out DName newDisplayName);
```

Comment:
This should remain internal. Likely, we should provide a default impl like DisabledDisplayNameProvider does. 

This particular call is only used by Canvas, and should not be exposed to other customers of PowerFx.

Reply:
Updated this.